config/jobs/GoogleCloudPlatform/k8s-cluster-bundle/k8s-cluster-bundle.yaml

```diff
@@ -0,0 +1,44 @@
+presubmits:
+  GoogleCloudPlatform/k8s-cluster-bundle:
+  - name: gcp-k8s-cluster-bundle-presubmit
```

Comment:
`pull-k8s-cluster-bundle-bazel-test` would be more consistent with the rest of our job names.

Reply:
Okay